web-api/src/business/useCases/health/getHealthCheckInteractor.ts

```diff
@@ -198,9 +198,7 @@ const getCognitoStatus = async ({
   applicationContext: ServerApplicationContext;
 }): Promise<boolean> => {
   try {
-    const cognito = applicationContext.getCognito();
-
-    await cognito.describeUserPool({
+    await applicationContext.getCognito().describeUserPool({
       UserPoolId: applicationContext.environment.userPoolId,
```

Comment:
Seems like this could also be behind the UserGateway? 

`.getUserGateway().getHealth(...)`